### ISSUE TITLE
Flume -  Cannot read property 'length' of null

### ISSUE THREAD

**Issue by evbogue** [ssbc/scuttlebot#368]
When using git-ssb to pull on a repo, the flume branch of sbot will quit out with this error

```
/home/ev/flume/node_modules/pull-write-file/index.js:17
          fs.write(fd, data, 0, data.length, pos, function (err, bytes) {
                                    ^

TypeError: Cannot read property 'length' of null
    at next (/home/ev/flume/node_modules/pull-write-file/index.js:17:37)
    at /home/ev/flume/node_modules/pull-stream/throughs/map.js:19:9
    at /home/ev/flume/node_modules/pull-stream/throughs/through.js:19:9
    at PacketStreamSubstream.weird.read (/home/ev/flume/node_modules/muxrpc/pull-weird.js:33:7)
    at PacketStream._onstream (/home/ev/flume/node_modules/packet-stream/index.js:188:12)
    at PacketStream.write (/home/ev/flume/node_modules/packet-stream/index.js:123:41)
    at /home/ev/flume/node_modules/muxrpc/pull-weird.js:56:15
    at /home/ev/flume/node_modules/pull-stream/sinks/drain.js:24:37
    at /home/ev/flume/node_modules/pull-stream/throughs/filter.js:17:11
    at Object.cb (/home/ev/flume/node_modules/packet-stream-codec/index.js:107:11)
```

**Comment by cryptix** [comment]
I think this was fixed in [pull-write-file@0.2.4](https://github.com/pull-stream/pull-write-file/commits/master).

**Comment by evbogue** [comment]
Yes, fixed.
